runtime/compiler/p/codegen/PPCJNILinkage.cpp

```diff
@@ -306,6 +306,10 @@ TR::Register *J9::Power::JNILinkage::buildDirectDispatch(TR::Node *callNode)
             }
          generateTrg1Src2Instruction(cg(), TR::InstOpCode::add, callNode, addrArg, addrArg, posArg);
 
+         if (crc32m2 || crc32m3) {
+            generateTrg1ImmInstruction(cg(), TR::InstOpCode::li, callNode, posArg, 0);
+         }
+
```

Comment:
1. Can you add a comment here to tell the reader that we re-order the parameters below and `posArg` will be bound to `gr6` which is the `castagnoli` parameter of the helper? Otherwise it is confusing why this is here.
2. Brace indentation needs fixing, or remove the braces.